### PR TITLE
Remove MultiQC as a submodule and instead clone as prebuild

### PR DESCRIPTION
**Pull request by vladsavelyev**
@ewels, next attempt!

### CODE REVIEW

**Review by ewels — COMMENTED** [review]
Looking great!

* Could we extend this to work with Wave
* Can we add it to `prebuild start` as well so that it runs locally for `npm run dev`

File: internal/fetch-docs-multiqc.mjs [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+// Clone the repository
+const tmpExists = await fs.pathExists(tmpPath);
+if (tmpExists) {
+  console.log("Cloned repo already exists, will pull latest changes");
+  await git(tmpPath).pull();
+  console.log("✅ Pulled latest changes");
+} else {
+  console.log(`Cloning MultiQC into ${tmpPath}`);
+  await git().clone(repoUrl, tmpPath);
```
Comment (jason-seqera) [inline internal/fetch-docs-multiqc.mjs:22]: What happens if this raises an exception? Should we be using `catch` on errors?

Comment (vladsavelyev) [reply internal/fetch-docs-multiqc.mjs]: Added a catch block

**Comment by jason-seqera** [comment]
Because this stays on disk and is updated, it looks like the `.gitignore` additions probably are necessary, or these repos will end up getting committed as submodules again.

**Comment by vladsavelyev** [comment]
Sorry, not following, what are you referring to? I am gitignoring the repos. And in any case, submodules won't be created automatically without explicit `git submodule init`.

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**